installer-scripts/SwiftFormat-amd64.wxs

```diff
@@ -0,0 +1,54 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+  <Package
+      Language="1033"
+      Manufacturer="https://github.com/nicklockwood/SwiftFormat"
```

Comment:
I think that this should be `nicklockwood` rather than the URL.

Reply:
Done